salt/modules/netmiko_mod.py

```diff
@@ -493,6 +493,7 @@ def exit_config_mode(**kwargs):
 def send_config(config_file=None,
                 config_commands=None,
                 template_engine='jinja',
+                commit=False,
```

Comment:
This looks like an API change -  can you add this to tend of the list, just in case?

Reply:
The netmiko module has been just added https://github.com/saltstack/salt/pull/48260 - there might be other upcoming changes if I notice anything misbehaving, sorry.

Reply:
Oh! I should have checked. Thanks! :)